extras/createdevproject.py

```diff
@@ -0,0 +1,47 @@
+"""
+Modifies the project template to fit with local development
+"""
+import os
+import sys
+from misago.core import setup
+
+
+PROJECT_NAME = 'forum'
+
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+def main():
+    global PROJECT_NAME, SETTINGS_FILE
+
+    # Allow for overriding project name
+    if len(sys.argv) > 1:
+        PROJECT_NAME = sys.argv[1]
+    else:
+        sys.argv.append(PROJECT_NAME)
+
+    settings_file = os.path.join(BASE_DIR, PROJECT_NAME, PROJECT_NAME, 'settings.py')
```

Comment:
Lets not create deep nesting for those files. Its fine to put `avatargallery`, `media`, `manage.py` etc. ect. in main dir, just make sure those are in `.gitignore` too.

Reply:
Not entirely sure if I understand. I'm just reading a settings file created by misago's `startproject` version.

Reply:
I'm saying that there should be additional steps involved:

`rm forum/cron.txt`
`rm -rf forum/static`
`mv forum/manage.py manage.py`
`mv forum/avatargallery avatargallery`
`mv forum/media media`
`rm -rf forum/theme`